KevDev90/Travel-Tracker
# Traveler styling

What's this PR do?
Adds functionality to get the date, displays a travelers trips on the dom, displays travelers name on the dom, adds initial styling to traveler and agent login pages, adds a mixin for header, makes the agent class extend the traveler class.

Where should the reviewer start?
By Looking at the newly built out methods and by also reviewing my class structure.

How should this be manually tested?
By pulling down the branch and viewing the changes

Any background context you want to provide?
I plan on still moving some index.js functionality either to traveler class or to a domupdates file sometime in the future. 

What are the relevant tickets?
Screenshots (if appropriate)
Questions:

Do you feel like my class structure is set up effectively? And if not what are some things I could possibly improve upon in that regard? Lmk if you see anything else that you feel should be changed or altered at this point, thanks!

@khalidwilliams
@hannahhch
